File: rest/incoming-phone-numbers/list-post-example-1/list-post-example-1.4.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

number = @client.account.incoming_phone_numbers
                .create(friendly_name: 'My Company Line',
                        voice_url: 'http://demo.twilio.com/docs/voice.xml',
                        phone_number: '+15105647903',
                        voice_method: 'GET')
puts number.sms_url

File: rest/incoming-phone-numbers/list-post-example-1/list-post-example-1.4.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Services_Twilio($sid, $token);

$number = $client->account->incoming_phone_numbers->create(
    array(
        "FriendlyName" => "My Company Line",
        "VoiceUrl" => "http://demo.twilio.com/docs/voice.xml",
        "PhoneNumber" => "+15105647903",
        "VoiceMethod" => "GET"
    )
);
echo $number->sid;
